boards/sparkfun/samd21_dev_breakout/doc/index.rst

```diff
@@ -0,0 +1,143 @@
+.. zephyr:board:: sparkfun_samd21_breakout
+
+Overview
+********
+
+The SparkFun SAMD21 dev breakout board is an arduino form factor development board.
+Designed to be a more powerful arduino. It shares the arduino form factor and gpio
+footprint. Based on the ATSAMD21G18 MCU, it runs a 32-Bit Arm Cortex-M0+ at up to 48MHz.
+More information can be found on the `SAMD21 dev breakout specification website`_.
+
+.. figure:: img/sparkfun_samd21_breakout.webp
+   :align: center
+   :alt: Sparkfun SAMD21 dev breakout board
+
```

Comment:
as mentioned before this needs to go -- check out https://builds.zephyrproject.io/zephyr/pr/95971/docs/boards/sparkfun/samd21_dev_breakout/doc/index.html you'll immediately understand why it's redundant (it is still showing the previous version of the page at the time I am writing this but this doesn't really matter) :)
"zephyr:board::" automagically adds the image to the sidebar on the right hand side

```suggestion
```

Reply:
Oh, I had no idea you could look at the docs per PR, that's really helpful. Okay, going to change this and push.